Mac Error

Trying to install on mac by the following commands:

pip install cython
pip install git+https://github.com/izderadicka/pdfparser

I received the following error:

   pdfparser/poppler.cpp:8741:41: error: assigning to 'char *' from incompatible type 'const char *'
            __pyx_t_12 = __pyx_v_font_name->getCString();
                         ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
    pdfparser/poppler.cpp:8840:29: error: assigning to 'char *' from incompatible type 'const char *'
        __pyx_t_12 = __pyx_v_s->getCString();
                     ~~~~~~~~~~~^~~~~~~~~~~~
    9 warnings and 2 errors generated.
    error: command 'clang' failed with exit status 1

    ----------------------------------------
Command "PATH/python2.7 -u -c "import setuptools, tokenize;__file__='....../pip-req-build-so5oPb/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record .....pip-record-bsmG7I/install-record.txt --single-version-externally-managed --compile --install-headers .../include/site/python2.7/pdfparser" failed with error code 1 in .....T/pip-req-build-so5oPb/


I've received this error on various of different macs.

Looking forward to your help,
Thanks

--- a/pdfparser/poppler.pyx
+++ b/pdfparser/poppler.pyx
@@ -24,7 +24,7 @@ cdef extern from "goo/GooString.h":
     cdef cppclass GooString:
         GooString(const char *sA)
         int getLength()
-        char *getCString()
+        const char *getCString()
         char getChar(int i)
 
 cdef extern from "OutputDev.h":

